README.md

```diff
@@ -395,6 +395,36 @@ When releasing pieces to the SDK our thought process is guided by the following
 
 - **How can I lowercase an email field anytime input is provided by a file or manual entry?**
   - This is a good use for field `compute`. This function will be idempotent (running it over and over on the same input produces the same output and state)
+- **Why can't I check for nulls in a `validate` function?
+You can't check for `null` or `undefined` in a validate function, because validate functions are never called with a value that isn't of the field's type.  The way to check for `null` is to set `required:true` on the field, this works for 95% of field use cases, this will flag an error at the review stage for fields provided with a `null` value.  Having this strict typing makes `validate` functions less error prone and less repetitious.  If we allowed `null` or `undefined` to propagate to `validate`,  every user provided function would have to start with checking for `null` or `undefined`, users who didn't do this would either see a typing error, or worse suffer unreliable code that was deployed.
+
+When would you want to check for `null` or `undefined` in a validate function?  The only place the built in `required` behavior doesn't work is when a `recordCompute` or `batchRecordsCompute` function was expected to provide a value for a field, and failed.  In that case, you are already writing a `recordCompute` or `batchRecordsCompute` function, check for the `null` there.
+
+We might add a `totalRequired` flag that checks that a field was filled out after `recordsCompute` and `batchRecordsCompute`.
```

Comment:
I worry about indicating we _may_ make a change down the road in case we don't end up doing that. It doesn't really provide anything actionable for users now.

Reply:
good point